velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -688,8 +690,81 @@ TpchPlan TpchQueryBuilder::getQ18Plan() const {
   return context;
 }
 
+TpchPlan TpchQueryBuilder::getQ19Plan() const {
+  static const std::string kLineitem = "lineitem";
+  static const std::string kPart = "part";
+  std::vector<std::string> lineitemColumns = {
+      "l_partkey",
+      "l_extendedprice",
+      "l_discount",
+      "l_shipinstruct",
+      "l_shipmode",
+      "l_quantity"};
+  std::vector<std::string> partColumns = {
+      "p_partkey", "p_brand", "p_container", "p_size"};
+
+  auto lineitemSelectedRowType = getRowType(kLineitem, lineitemColumns);
+  const auto& lineitemFileColumns = getFileColumnNames(kLineitem);
+  auto partSelectedRowType = getRowType(kPart, partColumns);
+  const auto& partFileColumns = getFileColumnNames(kPart);
+
+  auto planNodeIdGenerator = std::make_shared<PlanNodeIdGenerator>();
+  core::PlanNodeId lineitemScanNodeId;
+  core::PlanNodeId partScanNodeId;
+
+  std::string filterExpr =
+      " ((p_brand = 'Brand#12')"
+      " AND (l_quantity between 1.0 and 11.0)"
+      " AND ((p_container = 'SM CASE') OR (p_container = 'SM BOX') OR (p_container = 'SM PACK') OR (p_container = 'SM PKG'))"
```

Comment:
can we not use an IN clause here?

Reply:
No, using IN clause throws the following error:
parsing array literals requires a memory pool

Reply:
can you share the full backtrace?